src/@next/hooks/useLanguage.tsx

```diff
@@ -0,0 +1,15 @@
+import { I18nContext } from '@components/containers';
+import React from "react";
```

Comment:
To stick to the current convention we use, consider using this:
```
module imports
absolute imports
relative imports
```
with a new line in between